extensions/ecg-dicom/package.json

```diff
@@ -15,7 +16,7 @@
   "keywords": [
     "ohif-extension"
   ],
-  "main": "public/umd/@radicalimaging/ecg-dicom/index.umd.js",
+  "main": "dist/index.umd.js",
```

Comment:
If you map the dist/index.umd.js to http://host/umd/@packageName/ then it should work, but otherwise you will overlap items.